src/main/app/claims/models/claimantResponse.ts

```diff
@@ -0,0 +1,16 @@
+import { ResponseAcceptation } from 'claims/models/claimant-response/responseAcceptation'
+import { ResponseRejection } from 'claims/models/claimant-response/responseRejection'
+import { ClaimantResponseType } from 'claims/models/claimant-response/claimantResponseType'
+
+export type ClaimantResponse = ResponseAcceptation | ResponseRejection
+
+const deserializers = {
+  [ClaimantResponseType.ACCEPTATION]: ResponseAcceptation.deserialize,
```

Comment:
Suggesting `ACCEPTANCE` instead

Reply:
Acceptation matches what is used in the claim-store model so it is used here for consistency

Reply:
I think I'm with andy here. We can leave it as is for now as it matches the claim-store but I think we'll need to change it in both places later